docs/cql/cql-extensions.md

```diff
@@ -133,21 +133,34 @@ token ranges.
 
 ## Synchronous materialized views
 
-Materialized view updates can be applied synchronously (with errors propagated
-back to the user) or asynchronously, in the background. Historically, in order
-to use synchronous updates, the materialized view had to be local,
-which could be achieved e.g. by using the same partition key definition
-as the one present in the base table.
-Scylla also allows explicitly marking the view as synchronous, which forces
-all its view updates to be updated synchronously. Such views tend to reduce
-observed availability of the base table, because a base table write would only
-succeed if all synchronous view updates also succeed. On the other hand,
-failed view updates would be detected immediately, and appropriate action
-can be taken (e.g. pruning the materialized view, as mentioned in the paragraph
-above).
-
-In order to mark a materialized view as synchronous, one can use the following
-syntax:
+Usually, when a table with materialized views is updated, the update to the
+views happen _asynchronously_, i.e., in the background. This means that
+the user cannot know when the view updates have all finished - or even be
+sure that they succeeded.
+
+In Scyla (but not in Cassandra), in some special cases the view updates
```

Comment:
"special cases" sounds a bit confusing to me. Perhaps we could replace it with (and add a comma) "in some cases," or "in rare cases,"?

Reply:
Maybe "in some circumstances" would be clearer?
My point is that in ScyllaDB (sic), there are some circumstances - depending on how the table is set up - where the view updates were synchronous even before adding this feature. 
I thought the "special" word was useful because it emphasizes that this is not the usual case - the usual case was described in the previous paragraph (starting with "Usually, "). Maybe this isn't clear enough. However, I couldn't use words like "rare" because it isn't a matter of probability - your specific table either has this feature, or it doesn't. I'll try to change the wording a bit in the next version, take a look and see if you feel it's better, or have another proposal how to improve it.

Reply:
(by the way, I know that the adjective "special" experienced some semantic drift in English in recent years, but I used the phrase "special case" as is common in sciences - https://en.wikipedia.org/wiki/Special_case).

Reply:
I know why you used it, and it makes sense to me, but somehow it doesn't seem right in this context (perhaps because we tend to be super specific when talking about "special cases). Or I'm just nitpicking :)
Anyway, the new version is very clear to me and reads smoothly. Thank you!